cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -0,0 +1,134 @@
+The other branch contains a MapReduce program *TopNMapReduce* that consumes the normalized data and produces the top N words in the dataset *topn*
+
+Let's look at some of these components, and then run the application and see the results.
+
+The WikipediaPipeline Application
+---------------------------------
+
+As in the other `examples <index.html>`__, the components
+of the application are tied together by the class ``WikipediaPipelineApp``:
+
+.. literalinclude:: /../../../cdap-examples/WikipediaPipeline/src/main/java/co/cask/cdap/examples/wikipedia/WikipediaPipelineApp.java
+   :language: java
+   :lines: 24-55
+
+
+Running the Example
+===================
+
+.. highlight:: console
+
+Injecting data
+--------------
+The *pageTitleStream* consumes events in the format returned by the Facebook Likes Graph API.
+
+Once the application is deployed:
+
+  Inject a file of likes data to the stream *pageTitleStream* by running this command from the Standalone CDAP SDK
+directory, using the Command Line Interface::
+
+  $ cdap-cli.sh load stream pageTitleStream examples/WikipediaPipeline/data/fb-likes.txt
+  Successfully sent stream event to stream 'pageTitleStream'
+
+
+The *wikiStream* consumes events in the format returned by the MediaWiki Wikipedia API.
+
+Once the application is deployed:
+
+  Inject a file of points data to the stream *pageTitleStream* by running this command from the Standalone CDAP SDK
+  directory, using the Command Line Interface::
+
+  $ cdap-cli.sh load stream wikiStream examples/WikipediaPipeline/data/wikipedia-data.txt
+  Successfully sent stream event to stream 'wikiStream'
+
+Starting the Workflow
+---------------------
+
+There are three ways to start the *WikipediaPipelineWorkflow* program:
+
+#. Go to the *WikipediaPipeline* `application overview page
+   <http://localhost:9999/ns/default/apps/WikipediaPipeline/overview/status>`__,
+   click ``WikipediaPipelineWorkflow`` to get to the Workflow detail page, then click the *Start* button; or
+
+#. Send a query via an HTTP request using the ``curl`` command::
+
+    curl -w'\n' -v \
+      http://localhost:10000/v3/namespaces/default/apps/WikipediaPipeline/workflows/WikipediaPipelineWorkflow/start
+#. From the Standalone CDAP SDK directory, use the Command Line Interface::
+
+    $ cdap-cli.sh start workflow WikipediaPipeline.WikipediaPipelineWorkflow
+  
+    Successfully started flow 'WikipediaPipelineWorkflow' of application 'WikipediaPipeline' with stored runtime arguments '{}'
+
+You can also optionally specify the following runtime arguments for the *WikipediaPipelineWorkflow*:
+
+- *min.pages.threshold*: The threshold for number of pages to exist in the *pageTitleStream* for the Workflow to proceed. Defaults to 10.
+- *mode*: Set this to 'online' when you wish to download Wikipedia data over the internet. Defaults to offline, in which the Workflow expects
+Wikipedia data to be in the *wikiStream*

```

Comment:
This needs to be set up as this:
```
- *mode*: Set this to 'online' when you wish to download Wikipedia data over the internet. 
  Defaults to offline, in which the Workflow expects Wikipedia data to be in the *wikiStream*.
```
- Add trailing period
- Break lines so they fit the Github window
- Indent the second line so it matches the "*" and is then formatted correctly.